project/Component/NefCommon/Errors/RenderingOutput.swift

```diff
@@ -0,0 +1,21 @@
+//  Copyright © 2019 The nef Authors.
+
+import Foundation
+import Bow
+
+public struct RenderingOutput<A> {
+    public let ast: String
+    public let output: NEA<A>
+    
+    public init(ast: String, output: NEA<A>) {
+        self.ast = ast
+        self.output = output
+    }
+}
+
+
+public extension RenderingOutput {
+    typealias PageOutput  = RenderingOutput<A>
+    typealias PlaygroundOutput  = NEA<(page: RenderingURL, output: PageOutput)>
+    typealias PlaygroundsOutput = NEA<(playground: RenderingURL, output: PlaygroundOutput)>
```

Comment:
About your comments, yep, I did it to keep name consistency but I agree it could confuse you and make the code complex.

I have updated it:
```Swift
public extension RenderingOutput {
    typealias PlaygroundOutput  = NEA<(page: RenderingURL, output: RenderingOutput<A>)>
    typealias PlaygroundsOutput = NEA<(playground: RenderingURL, output: PlaygroundOutput)>
}
```

So, we do not need to parametrize to `A`; and it will be consumed in the modules without lose the model information

```Swift
typealias RenderingOutput = NefCommon.RenderingOutput<String>
typealias PlaygroundOutput  = RenderingOutput.PlaygroundOutput
typealias PlaygroundsOutput = RenderingOutput.PlaygroundsOutput
```